src/components/Homepage/events/getEvents.js

```diff
@@ -1,9 +1,12 @@
 import { format, isAfter } from 'date-fns'
 
+const toDate = stringDate => new Date(stringDate)
+
 export const getHomepageMeetups = events =>
   events
     .filter(n => !n.node.homepageFeatured)
     .filter(n => isAfter(new Date(n.node.date), new Date()))
+    .sort((a, b) => (toDate(a.node.date) < toDate(b.node.date) ? -1 : 1))
```

Comment:
Minor edge case: if `a.node.date` and `b.node.date` are equal, this will always return `-1` meaning that if an update occurred that causes a re-render of the component these are stored in, the values will switch places over and over. try to always use `<=` or `>=` in comparator functions / lambdas.

Reply:
Great point. I didn't think of it but it would switch them for sure. I used `<=`. Thanks for  pointing it out